app/models/decidim/kids/minor_account.rb

```diff
@@ -16,6 +16,13 @@ class MinorAccount < ApplicationRecord
       validate :can_be_tutor
       validate :can_be_minor
 
+      def promote_account
```

Comment:
```suggestion
      def promote_account!
```
As this is destructive, maybe this convention would be better

Reply:
Done